### DISCUSSION TITLE
Information icons next to keys?

### DISCUSSION THREAD

**Discussion by blockninja124** [misode/misode.github.io#627] [Q&A] [answered]
So I noticed recently that on some pages, theres a little info icon next to stuff that gives you a little explanation. However I was unable to find any of these on pages I could find the source of. Is there a way to do this in a custom page, and where could I have found the code that does this?
![image](https://github.com/user-attachments/assets/0de7cc4e-d8ac-44a7-8b30-1feb4bcb2379)
(from https://misode.github.io/loot-table/)

**Comment by misode** [comment, marked as answer]
The information icon will check if a translation key exists that matches the field key + `.help`. So for instance `loot_pool.rolls.help`

**Reply by blockninja124** [reply]
Thanks!
